src/component/visualMap/VisualMapModel.ts

```diff
@@ -247,6 +257,30 @@ class VisualMapModel<Opts extends VisualMapOption = VisualMapOption> extends Com
      * @return An array of series indices.
      */
     protected getTargetSeriesIndices(): number[] {
+        const seriesTargets = this.option.seriesTargets;
+        if (seriesTargets) {
+            // When seriesTargets is provided, collect all target series indices
+            const indices: number[] = [];
+            for (const target of seriesTargets) {
```

Comment:
There seems to be no `const of` usage for the array iteration in the codebase. As the current coding style, maybe the `zrUtil.each` utility should be used here.

Reply:
done